aws/resource_aws_emr_cluster_test.go

```diff
@@ -3448,11 +3528,13 @@ resource "aws_emr_cluster" "test" {
 
   master_instance_group {
     instance_type = "m4.large"
+    market        = "ON_DEMAND"
```

Comment:
Please remove changes to existing test configurations so we can verify no regressions are being introduced.